infra/gcp/istio-prow-private/cluster.tf

```diff
@@ -0,0 +1,75 @@
+module "prow" {
```

Comment:
There are lots of things named "prow". might i suggest something else like 
- prow-cluster
- prow-gke-cluster
- prow-primary